src/controllers/dynakube/oneagent/daemonset/env_vars.go

```diff
@@ -83,20 +126,6 @@ func mapToArray(envVarMap map[string]corev1.EnvVar) []corev1.EnvVar {
 	return result
 }
 
-func (dsInfo *builderInfo) setDefaultProxy(envVarMap map[string]corev1.EnvVar) map[string]corev1.EnvVar {
-	if dsInfo.instance.Spec.Proxy.ValueFrom != "" {
-		setDefaultValueSource(envVarMap, proxy, &corev1.EnvVarSource{
-			SecretKeyRef: &corev1.SecretKeySelector{
-				LocalObjectReference: corev1.LocalObjectReference{Name: dsInfo.instance.Spec.Proxy.ValueFrom},
-				Key:                  "proxy",
-			},
-		})
-	} else {
-		setDefaultValue(envVarMap, proxy, dsInfo.instance.Spec.Proxy.Value)
-	}
-	return envVarMap
-}
-
 func setDefaultValue(envVarMap map[string]corev1.EnvVar, name string, value string) map[string]corev1.EnvVar {
```

Comment:
should this (and the Source one) be in `kubeobjects.env.go`?

Reply:
probably, but I think the PR is big enough already to move things around more